packages/main/src/plugin/color-registry.ts

```diff
@@ -1000,6 +1000,10 @@ export class ColorRegistry {
       dark: colorPalette.white,
       light: colorPalette.black,
     });
+    this.registerColor(`${button}close-hover-bg`, {
+      dark: colorPalette.white + '2',
+      light: colorPalette.black + '2',
```

Comment:
+2 ?

Reply:
Makes it translucent (3 char rgb + an 'a' value). Not great, but we already had to use the same in Link.

Reply:
It same as the hover `Link` component as proposed by Tim